packages/devtools_app/lib/src/timeline/timeline_flame_chart.dart

```diff
@@ -262,59 +258,60 @@ class FullTimelineFlameChartCanvas extends FlameChartCanvas<FullTimelineData> {
       }
     }
 
-    int currentRow = 0;
-    int currentSection = 0;
+    int currentRowIndex = 0;
+    int currentSectionIndex = 0;
 
     for (String bucketName in data.eventBuckets.keys) {
+      final List<TimelineEvent> bucket = data.eventBuckets[bucketName];
+      int maxBucketDepth = 0;
```

Comment:
nit: rename maxBucketDepth to just be
sectionDepth as it looks like it is the exact depth of the section.